### PR TITLE
Ana/fix endpoint should start with ws

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Trying to fix this. But only hacky fix for now. I don't know why `polkadotNetwork.rpc_url` is `null` sometimes:

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f6a238950882b013db0958e--lunieio.netlify.app

**Comment by faboweb** [comment]
let's close until you can reproduce

**Comment by Bitcoinera** [comment]
I think this happens when changing branches. Going to check

**Comment by Bitcoinera** [comment]
Exactly, this error happens every time branch is changed or API gets redeployed. Still investigating why

**Comment by Bitcoinera** [comment]
Also fixes this minor style glitch:

Before:

image


Now:

image

File: api/lib/resolvers.js [hunk 1]
```diff
@@ -280,7 +280,7 @@ const resolvers = (networkList, notificationController) => ({
         .map((network) => {
           // the server side nodes are mostly whitelisted or secret so we don't want to send them to the FE
           return Object.assign({}, network, {
-            rpc_url: network.public_rpc_url,
+            rpc_url: network.public_rpc_url || network.rpc_url,
```
Comment (faboweb) [inline api/lib/resolvers.js:283]:
```suggestion
            rpc_url: network.public_rpc_url
```

the comments states that we don't want to pass the private rpc out

Comment (Bitcoinera) [reply api/lib/resolvers.js:283]: there are some networks in staging-DB that lack of `rpc_url`, like cosmos for example. Guess we could use nylira there, but what about the others :shrug:

Comment (faboweb) [reply api/lib/resolvers.js:283]: is this a problem? you only had the issue with Polkadot, right?

Comment (Bitcoinera) [reply api/lib/resolvers.js:283]: yeah, true. Which left me wondering. OK, let me try this PR without that bit and see if it still works

Comment (Bitcoinera) [reply api/lib/resolvers.js]: Yeah, it works flawlessly. So the real fix was just adding `public-rpc-url` to the query body actually

**Comment by Bitcoinera** [comment]
ping. Let's merge! (Changelog for `/app` not needed. Merely added a margin top for the Rewards message for Polkadot)

**Review by jbibla — APPROVED** [review]

**Pull request merged**